### PR TITLE
Infrastructure: Fix failing `npm run regression-report` script

### PR DESCRIPTION
**Pull request by howard-e**
Address comments in #959.

cc @nschonni 
___
[WAI Preview Link](https://deploy-preview-41--wai-aria-practices2.netlify.app)

### CODE REVIEW

**Comment by a11ydoer** [comment]
@howard-e it may be the separate issue but I have error of **missing script:"regression-report"** when I did npm run regression-report. When I check the available list of script using "npm run", fmt, lint, and boostrap were only available. I am mentioning @spectranaut in case she can help.

**Comment by howard-e** [comment]
@a11ydoer when I do `npm run`, I get the following:

<details>
<summary>npm run result</summary>

```sh
❯ npm run
Lifecycle scripts included in aria-practices@0.0.0:
  test
    npm run lint && npm run regression

available via `npm run-script`:
  fix
    npm run lint:es -- --fix && stylelint --fix  "**/*.css" && prettier --write "**/*.css"
  htmlhint
    htmlhint "**/*.html" "**/*.template" --ignore "common/**/*.html" --format unix
  lint
    npm run lint:es && npm run lint:css && npm run lint:html && npm run lint:spelling
  lint:css
    stylelint "**/*.css" && prettier --check "**/*.css"
  lint:es
    eslint . --report-unused-disable-directives
  lint:js
    npm run lint:es
  lint:html
    npm run htmlhint && npm run vnu-jar
  lint:spelling
    cspell "**/*.*"
  link-checker
    node ./scripts/link-checker.js
  reference-tables
    node scripts/reference-tables.js
  regression
    ava --timeout=1m
  regression-report
    node test/util/report
  vnu-jar
    java -jar node_modules/vnu-jar/build/dist/vnu.jar --filterfile .vnurc --no-langdetect --skip-non-html content/
  create-gh-project
    node ./scripts/create-gh-project.js
  prepare
    husky install
```

</details>

I'm not familiar with the `fmt` and `bootstrap` commands in this repo. Is it possible you may have ran that command in a separate repo's project?

**Comment by a11ydoer** [comment]
sorry. you are right. this is downside of multi tasking. I was using old repo not updated one.

File: test/util/report.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const fs = require('fs');
 const htmlparser2 = require('htmlparser2');
 const spawn = require('cross-spawn');
 
-const examplePath = path.resolve(__dirname, '..', '..', 'examples');
+const examplePath = path.resolve(__dirname, '..', '..', 'content', 'patterns');
```
[the remark this answers is not in the document]
Comment (a11ydoer) [reply test/util/report.js:10]: @howard-e I confirmed that the path update here works. Thanks again!

**Review by a11ydoer — APPROVED** [review]
"npm run regression-report" is working due to directory path update.

**Comment by mcking65** [comment]
@howard-e 

I am getting a runtime error as follows.

~~~
npm run regression-report

aria-practices@0.0.0 regression-report
node test/util/report
C:\Users\mck\git\APGRepo\main\test\util\report.js:208
    exampleCoverage[example].missingTests.delete(matchResults[2]);
                             ^
TypeError: Cannot read properties of undefined (reading 'missingTests')
    at getRegressionTestCoverage (C:\Users\mck\git\APGRepo\main\test\util\report.js:208:30)
    at Object.<anonymous> (C:\Users\mck\git\APGRepo\main\test\util\report.js:217:1)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47
Node.js v18.12.0
~~~

**Comment by howard-e** [comment]
> @howard-e
> 
> I am getting a runtime error as follows.

@mcking65 Oops! Some cross-platform support (for Windows) was missed there. https://github.com/w3c/aria-practices/pull/2291/commits/0423ad708dda7d0c07da067c342b6e86c536cf83 should address this.

**Review by mcking65 — APPROVED** [review]
Thank you @howard-e! This is now working.

**Pull request merged**